CMakeSettings.json

```diff
@@ -0,0 +1,146 @@
+{
+  // See https://go.microsoft.com//fwlink//?linkid=834763 for more information about this file.
+  "configurations": [
+    {
+      "name": "x86-Debug",
```

Comment:
Should we remove these other configurations (those besides x64-Debug) until they are functional?